pkgs/applications/audio/csound/csound-manual/default.nix

```
@@ -1,18 +1,19 @@
-{ 
-  stdenv, fetchurl, docbook_xsl,
-  docbook_xml_dtd_45, python, pygments, 
-  libxslt 
+{
+  stdenv, fetchFromGitHub, docbook_xsl,
+  docbook_xml_dtd_45, python, pygments,
+  libxslt
 }:
 
 stdenv.mkDerivation rec {
   version = "6.12.0";
   name = "csound-manual-${version}";
-  
-  src = fetchurl {
-    url = "https://github.com/csound/manual/archive/${version}.tar.gz";
-    sha256 = "1v1scp468rnfbcajnp020kdj8zigimc2mbcwzxxqi8sf8paccdrp";
-  };
 
+  src = fetchFromGitHub {
+    owner = "csound";
+    repo = "manual";
+    rev = "3b0bdc83f9245261b4b85a57c3ed636d5d924a4f";
```

```suggestion
    rev = version;
```
If you use the tagged git version instead of the SHA hash, the derivation will be more amenable to automated updating.

---

Yes, I've asked the maintainers to do a release. I'll push harder for it maybe. Will check back tomorrow or this weekend, otherwise this can be merged like this (with pname and version as rev)

---

You don't need a release, there is already a tag for 6.12.0.

---

Oh, I guess you are concerned that 6.12.0 isn't considered stable without a release. That's a good point.

---

There are a lot of projects which don't have their tags edited on GitHub (only then it's considered a "release" by GitHub).

---

Ok let's do a revision version until a new tag/release is made, also curious what happens in terms of preferability, how I understand nix, it shouldn't matter.